rpc/wrtc_signaling_server.go

```diff
@@ -508,5 +523,7 @@ func (srv *WebRTCSignalingServer) OptionalWebRTCConfig(
 // Close cancels all active workers and waits to cleanly close all background workers.
 func (srv *WebRTCSignalingServer) Close() {
 	srv.cancelFunc()
-	srv.activeBackgroundWorkers.Wait()
+	srv.callMu.Lock()
+	defer srv.callMu.Unlock()
+	srv.callWorkers.Wait()
```

Comment:
super minor - i noticed we `Wait -> Unlock` without a `defer` in the corresponding [caller PR](https://github.com/viamrobotics/goutils/pull/262/files#diff-5514904bc577d9192b1f4805ade57d369a09c437b0bc2036986efa4be3b2bd0bR230-R233). any reason this one should be different (or vice versa)?

Reply:
Not intentional! I'll standardize the other PR to a `defer` :) good catch.